keylime/ca_util.py

```diff
@@ -63,15 +63,22 @@ def load_cert_by_path(cert_path: str) -> Certificate:
     return cert
 
 
-def setpassword(pw: Optional[str]) -> None:
+def setpassword(key_store_pw: Optional[str]) -> None:
     global global_password
-    if not pw:
-        pw = getpass.getpass("Please enter the password to decrypt your keystore: ")
+    if not key_store_pw:
+        key_store_pw = config.get("ca", "password")
+        if key_store_pw:
+            if key_store_pw == "default":
+                if logger:
+                    logger.warning("Using 'default' password option from CA configuration file")
+            key_store_pw = str(key_store_pw)
```

Comment:
No need? The `config.get` already return a string (per the signature)

Reply:
Again, the idea was the keep the messages emitted by the different components close to the "original behavior"